e2e/gui/src/test/java/com/epam/pipeline/autotests/DavMountStorageLimitTest.java

```diff
@@ -0,0 +1,339 @@
+                .showMetadata()
+                .disableFileSystemAccessIfNeeded();
+            library()
+                .selectStorage(storage1)
+                .clickEditStorageButton()
+                .clickOnPermissionsTab()
+                .deleteIfPresent(userRoleGroup)
+                .closeAll();
+        }
+    }
+
+    @Test
+    @TestCase(value = {"2233_2"})
+    public void checkMaxNumberOfStoragesWithWebdavAccess() {
+        final String[] davMountMaxStoragesInitial = navigationMenu()
+                .settings()
+                .switchToPreferences()
+                .getLinePreference(storageDavMountMaxStorages);
+        try {
+            navigationMenu()
+                    .settings()
+                    .switchToPreferences()
+                    .setPreference(storageDavMountMaxStorages, "2", true)
+                    .saveIfNeeded();
+            Stream.of(storage1, storage2)
+                .forEach(stor -> {
+                            library()
+                                    .selectStorage(stor)
+                                    .showMetadata()
+                                    .click(FILE_SYSTEM_ACCESS)
+                                    .sleep(5, SECONDS);
+                        });
+            library()
+                    .selectStorage(storage3)
+                    .showMetadata()
+                    .click(FILE_SYSTEM_ACCESS)
+                    .messageShouldAppear(errorMessage);
+        } finally {
+            logout();
+            loginAs(admin);
+            navigationMenu()
+                    .settings()
+                    .switchToPreferences()
+                    .setPreference(storageDavMountMaxStorages,
+                            davMountMaxStoragesInitial[0],
+                            parseBoolean(davMountMaxStoragesInitial[1]))
+                    .saveIfNeeded();
+            Stream.of(storage1, storage2)
+                    .forEach(stor ->
+                            library()
+                                    .selectStorage(stor)
+                                    .showMetadata()
+                                    .disableFileSystemAccessIfNeeded());
+        }
+    }
+
+    @Test
+    @TestCase(value = {"2233_3"})
+    public void checkDisableLink() {
+        try {
+            library()
+                    .selectStorage(storage1)
+                    .clickEditStorageButton()
+                    .clickOnPermissionsTab()
+                    .addNewUser(user.login)
+                    .selectByName(user.login)
+                    .showPermissions()
+                    .set(READ, ALLOW)
+                    .set(WRITE, ALLOW)
+                    .closeAll();
+            library()
+                    .selectStorage(storage1)
+                    .showMetadata()
+                    .click(FILE_SYSTEM_ACCESS)
+                    .ensure(FILE_SYSTEM_ACCESS, text("File system access enabled till"))
```

Comment:
"File system access enabled till" can be introduced a constant

Reply:
Done